libenkf/src/enkf_main_jobs.c

```diff
@@ -538,21 +538,45 @@ void * enkf_main_analysis_update_JOB( void * self , const stringlist_type * args
   return NULL;
 }
 
-
+// SHould skip compltely if data_root is unset.
 void * enkf_main_pre_simulation_copy_JOB( void * self , const stringlist_type * args) {
   const char * source_path  = stringlist_iget( args , 0 );
```

Comment:
Perhaps raise or at least log an error if the length of args is not 1 or 2?

Reply:
Length 1 is actually perfectly OK - but I have added log calls for the copy operation which is done.